.travis.yml

```diff
@@ -1,19 +1,24 @@
 sudo: required
+
 services:
 - docker
-language: node_js
-node_js:
-- node
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
 script:
-- npm run lint
-- npm run snyk-test
-- npm run generate-coverage
-- npm run check-coverage
-- npm run upload-coverage-coveralls
+  - docker-compose -f docker-compose-travis.yml up --build -d
```

Comment:
I can see the use of `-d` is done because the build hangs on Travis if not (only because I've just tested it via https://github.com/nhsuk/nearby-services-api/pull/40). However, we need to run it so that the build will fail when the command returns with a non 0 exit code. In that linked PR, the command did fail and I suspect that is what is happening with this build but it isn't being reported 🤔